### ISSUE TITLE
Game crashes before title screen

### ISSUE THREAD

**Issue by piecheese11** [Zygahedron/Starbecue#66]
[starbound.log](https://github.com/Zygahedron/Starbecue/files/11115651/starbound.log)
Reading the crash log (which I have attached the full copy of,) the following error appears to cause the crash:
**[Error] Application: exception thrown, shutting down: (StatusEffectDatabaseException) Duplicate stat effect named 'speciesAnimOverride', config file '/stats/speciesAnimOverride/speciesAnimOverride.statuseffect'**
What's causing this? I tried installing the SpeciesAnimOverrides in case that was a required mod that you didn't mention, but it didn't help.

**Comment by Zygahedron** [comment]
First off, the release page does mention SpeciesAnimOverrides as a requirement, make sure you read that properly. Also, it seems you already had it, as now you have it installed twice (once of an unknown version as a folder, and once as a .pak of v0.3).  
The crash however, is caused by you having both Starbecue / SpeciesAnimOverrides *and* the SBQ Uninstall mod. The uninstall one is only necessary if you wish to stop using the mod, and should be used on its own without Starbecue or SpeciesAnimOverrides.  
Additionally, Quickbar Mini is deprecated, as it's included in Stardust Core.

In summary, remove the following files/folders from your mods folder:
- `SBQ-uninstall.pak`
- `starboundSpeciesAnimOverrides-main`
- `QuickbarMini.pak`
